app/templates/index.hbs

```diff
@@ -163,97 +143,84 @@
   <div class="layout container">
     <div class="lg:col-4 lg:start-2 text-center">
       <a href="https://emberconf.com" class="es-button mb-1">
-        Learn about EmberConf
+        {{t "index.community.emberconf"}}
       </a>
       <LinkTo @route="community.meetups" class="es-button-secondary ml-1">
-        Ember Meetups
+        {{t "index.community.meetups"}}
       </LinkTo>
     </div>
 
     <div class="lg:col-4 lg:start-2 my-3 text-center">
-      <h3 class="text-lg">
-        Our official communication channels to get in touch with the community
-      </h3>
+      <h3 class="text-lg">{{t "index.communication.title"}}</h3>
     </div>
 
     <ul class="grid lg:grid-2 list-unstyled">
       <EsLinkCard
         class="lg:col-3"
         @icon="discourse"
-        @title="Discussion forum"
+        @title={{t "index.communication.forum.title"}}
         @href="http://discuss.emberjs.com"
       >
-        <p>Post and search longer-form questions in our public forum.</p>
+        <p>{{t "index.communication.forum.sub"}}</p>
       </EsLinkCard>
 
       <EsLinkCard
         class="lg:col-3"
         @icon="discord"
-        @title="Discord community chat"
+        @title={{t "index.communication.discord.title"}}
         @href="https://discord.gg/emberjs"
       >
-        <p>Join our real-time chat server to connect with other developers and get answers.</p>
+        <p>{{t "index.communication.discord.sub"}}</p>
       </EsLinkCard>
     </ul>
 
-    <p class="lg:col-4 lg:start-2 my-3 text-center">
-      Beyond our official core online channels, you can dig deeper with these additional learning resources from enthusiastic members of our
-      welcoming community!
-    </p>
+    <p class="lg:col-4 lg:start-2 my-3 text-center">{{t "index.learning.sub"}}</p>
 
     <ul class="list-unstyled layout">
       <EsCard
         class="lg:col-2 text-center"
         @icon="podcasts"
       >
-        <h4 class="text-md">Podcasts</h4>
+        <h4 class="text-md">{{t "index.learning.podcasts.title"}}</h4>
 
         <ul class="list-unstyled">
-          <li><a href="https://emberweekend.com/episodes">Ember Weekend</a></li>
-          <li><a href="https://embermap.com/podcast">Ember Map Podcast</a></li>
+          {{t "index.learning.podcasts.list" htmlSafe=true}}
```

Comment:
By putting the whole list in a translation, different languages can either 
 - swap these with links to the same resources in a different language (maybe that's supported via query params or something), 
 - add language-specific resources
 - replace all the links entirely